ext/mysqli/mysqli_api.c

```diff
@@ -2254,7 +2254,7 @@ PHP_FUNCTION(mysqli_ssl_set)
 		}
 	}
 
-	mysql_ssl_set(mysql->mysql, ssl_parm[0], ssl_parm[1], ssl_parm[2], ssl_parm[3], ssl_parm[4]);
+	mysql_ssl_set(mysql->mysql, ssl_parm[0], ssl_parm[1], ssl_parm[2], ssl_parm[3], ssl_parm[4], NULL);
```

Comment:
Sorry for the late response. I'm thinking about the possibility to use some #ifdef some-compile-time-flag #endif to wrap it. I'm not sure about if there's also a scenario pdo_mysql used with libmysql, or any other similar case. Also I'm not sure about the coding style. Will look into it further tomorrow.